lib/saving_account.rb

```diff
@@ -0,0 +1,25 @@
+module Bank
+   require "./account.rb"
+  class SavingsAccount < Account
+
+
+    def initialize(id, date)
+      super
```

Comment:
When you are reusing the duplicate parent method, you don't need to specify it. So if you just left out the `initialize` method here you would get the same result.